src/tokens.ts

```diff
@@ -102,6 +102,15 @@ export const TOKEN_SYMBOLS_MAP = {
       [CHAIN_IDs.MODE_SEPOLIA]: "0x4200000000000000000000000000000000000006",
     },
   },
+  LSK: {
+    name: "Lisk",
+    symbol: "LSK",
+    decimals: 18,
+    addresses: {
+      [CHAIN_IDs.LISK]: "0xac485391EB2d7D88253a7F1eF18C37f4242D1A24",
+      [CHAIN_IDs.MAINNET]: "0x6033f7f88332b8db6ad452b7c6d5bb643990ae3f"
```

Comment:
Awaiting confirmation from Ryan on this address, since I sourced it from CoinGecko.

Reply:
Confirmed here: https://github.com/LiskHQ/lisk-contracts/blob/main/deployment/addresses/mainnet/addresses.md